sources/core/dd-tree-menu-item.html

```diff
@@ -6,14 +6,14 @@
             <!-- ko if: hasItems -->
             <div class="c-items-menu-collapser"
                  data-bind="click: collapserClick, clickBubble: false, css: { 'add-element' : !hasItems }">
-                <svg class="button-svg-icon">
```

Comment:
Probably we need to have our prefix for classes like `button...` and `action...`. I believe we should use our `abris-` prefix for all our classes. The thing is that `button` and `action` class names can intersect with 3rd party libaries.

But I'm not sure we need to do it immediately.